ff_derive/src/lib.rs

```diff
@@ -538,7 +538,7 @@ fn prime_field_constants_and_sqrt(
             };
 
             quote! {
-                // Tonelli-Shank's algorithm for q mod 16 = 1
+                // Tonelli-Shank's algorithm works for every odd prime.
```

Comment:
This should say: "Tonelli-Shank's algorithm works for every remaining odd prime (i.e. q = 1 (mod 4))."